2025_metadata.json

```diff
@@ -66,6 +66,12 @@
     "description": "Redux Robotics 2025 Beta",
     "website": "https://docs.reduxrobotics.com/reduxlib"
   },
+  {
+    "uuid": "1ccce5a4-acd2-4d18-bca3-4b8047188400",
+    "name": "Yet Another Generic Swerve Library (YAGSL)",
+    "description": "YAGSL (Yet Another Generic Swerve Library) is an open-source robotics library designed to simplify and optimize the control, simulation, and configuration of swerve drive systems of all types.",
+    "website": "https://docs.yagsl.com/"
```

Comment:
Since YAGSL requires additional configuration beyond just installing the vendordep, it might benefit from adding an instructions key, which will open a website in VS Code.